components/Portfolio/Trading/Overview/index.tsx

```diff
@@ -21,17 +21,19 @@ import { LabelledTracers } from '@libs/types/TracerTypes';
 
 interface OProps {
     tracers: LabelledTracers;
-    positions: Tracer[];
-    holdings: Tracer[];
+    fetchedTracers: Tracer[];
```

Comment:
I think passing in both of these is a bit over kill, they should both be the same

Reply:
It's just fetchedTracers now